tests/Behat/Element/Admin/OrderCreateFormElement.php

```diff
@@ -80,13 +80,39 @@ public function specifyBillingAddress(AddressInterface $address): void
         );
     }
 
-    public function selectShippingMethod(string $shippingMethodName): void
+    public function getAvailableShippingMethods(): array
     {
         $this->clickOnTabAndWait('Shipments & Payments');
 
         $shipmentsCollection = $this->getDocument()->find('css', '#sylius_admin_order_creation_new_order_shipments');
 
+        if (count($shipmentsCollection->findAll('css', '[data-form-collection="item"]')) === 0) {
+            $shipmentsCollection->clickLink('Add');
+        }
+
+        $this->waitForFormToLoad();
+
+        $shippingMethods = $this->getDocument()->findAll('css', sprintf(
+            '#sylius_admin_order_creation_new_order_shipments [data-form-collection="item"]:last-child select option'
```

Comment:
sprintf not needed since you don't provide any values to be injected into the string